tests/unit/flow-orchestrate/test_flow_routing.py

```diff
@@ -32,10 +32,10 @@ def test_expected_messages_routing():
     )
 
     with f:
-        results = f.post(on='/index', inputs=[Document(text='1')], return_results=True)
+        docs = f.post(on='/index', inputs=[Document(text='1')], return_results=True)
         # there merge executor actually does not merge despite its name
-        assert len(results[0].docs) == 2
-        assert results[0].docs[0].text == 'merged'
+        assert len(docs) == 2
```

Comment:
try moving outside context manager

Reply:
Ok but a lot of tests have the assert inside the context manager; I left it where it is and just changed it. Should I go through all the existing tests and move the asserts outside the context manager, while I'm at it already?

Reply:
Just move them as u see, do not focus on this, but if u see it while changing and remember is nice to change